### PR TITLE
TemplateAddFragment Refactor

### PR DESCRIPTION
**Pull request by xvepkj**
### Summary
- Refactored TemplateAddFragment

### Refactoring Naming Doc
- [Google DOC](https://docs.google.com/document/d/1LrCTI5NWcVq4as1ZT4vqMgu9WatrJqkynuMtXVTFHbM/edit?usp=sharing)

### CODE REVIEW

File: app/src/main/java/com/xve/scheduler/ui/templates/add/TemplateAddFragment.kt [hunk 1]
```diff
@@ -129,25 +93,27 @@ class TemplateAddFragment : Fragment() {
...
+
       if (event.eventType != EventType.UNTRACKED) {
-        tagColorView.visibility = View.VISIBLE
+        tagColorView.visibility = VISIBLE
         var tag = tagsViewModel.get(event.tagId)
         if (!tag.isActive) tag = tagsViewModel.get(0)
-        tagColorView.setColorFilter(
-          tag.color,
-          PorterDuff.Mode.SRC_ATOP)
+        tagColorView.setColorFilter(tag.color, PorterDuff.Mode.SRC_ATOP)
```
Comment (gnaagar) [inline app/src/main/java/com/xve/scheduler/ui/templates/add/TemplateAddFragment.kt:102]: really?

File: app/src/main/java/com/xve/scheduler/ui/templates/add/TemplateAddFragment.kt [hunk 2]
```diff
@@ -129,25 +93,27 @@ class TemplateAddFragment : Fragment() {
...
-      if(event.eventType == EventType.TRACKED){
-        trackedEventCheckbox.visibility = VISIBLE
-        trackedEventCheckbox.isEnabled = false
-      }
-      if(event.eventType == EventType.LOGGED){
-        loggedProgress.text = "0.0"
-        loggedProgress.visibility = VISIBLE
-        loggedProgress.isEnabled = false
-      }
+      if(event.eventType == EventType.TRACKED)
```
Comment (gnaagar) [inline app/src/main/java/com/xve/scheduler/ui/templates/add/TemplateAddFragment.kt]: use `when`

**Review by gnaagar — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/xve/scheduler/ui/templates/add/TemplateAddFragment.kt (hunk 1 above)
Comment (xvepkj) [reply app/src/main/java/com/xve/scheduler/ui/templates/add/TemplateAddFragment.kt:102]: Yes

File: app/src/main/java/com/xve/scheduler/ui/templates/add/TemplateAddFragment.kt (hunk 2 above)
Comment (xvepkj) [reply app/src/main/java/com/xve/scheduler/ui/templates/add/TemplateAddFragment.kt]: Done

**Review by gnaagar — APPROVED** [review]

**Pull request merged**